addons/web/static/tests/views/list_view_tests.js

```diff
@@ -15691,4 +15691,19 @@ QUnit.module("Views", (hooks) => {
             assert.hasClass(target.querySelectorAll(".o_data_row")[1], "o_selected_row");
         }
     );
+
+    QUnit.test("no highlight of a column without label", async function (assert) {
+        await makeView({
+            type: "list",
+            resModel: "foo",
+            serverData,
+            arch: `
+                <tree default_order="foo">
+                    <field name="foo" nolabel="1"/>
+                </tree>
+            `,
+        });
+        assert.containsOnce(target, "thead th[data-name=foo]");
+        assert.doesNotHaveClass(target.querySelector("thead th[data-name=foo]"), "table-active");
```

Comment:
maybe add a field with `nolabel="0"` and assert this one has the table-active class ?
asserting something is not there isn't safe enough IMHO

Reply:
done